lib/jxl/enc_frame.cc

```diff
@@ -1173,6 +1372,25 @@ Status EncodeFrame(const CompressParams& cparams_orig,
                           ib.alpha(), lossless);
       }
     }
+    if (ApplyOverride(cparams.separate,
```

Comment:
For now, I wouldn't use the --separate heuristic by default in any case.

Reply:
It's `kOff` by default.

Reply:
Ah, true - I forgot that kOff is not kDefault ;)